src/neuroconv/datainterfaces/ophys/suite2p/suite2pdatainterface.py

```diff
@@ -47,6 +47,18 @@ class Suite2pSegmentationInterface(BaseSegmentationExtractorInterface):
     associated_suffixes = (".npy",)
     info = "Interface for Suite2p segmentation."
 
+    @classmethod
+    def get_source_schema(cls) -> dict:
+        schema = super().get_source_schema()
+        schema["properties"]["folder_path"][
+            "description"
+        ] = "Path to the folder containing Suite2p segmentation data. Should contain 'plane#' subfolder(s)."
+        schema["properties"]["plane_name"][
+            "description"
+        ] = "The name of the plane to load. This interface only loads one plane at a time. Use the full name, e.g. 'plane0'. If this values is omitted, the first plane found will be loaded."
```

Comment:
Was it missing the argument entirely or just a lackluster description?

Reply:
It didn't have a description at all